From 9bb4b35ea4a708b4740057fd4bce812407a2743d Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 4 Apr 2011 22:46:30 -0500
Subject: fix-flymake-xmlstarlet-invocation.diff
 * Emacs now invokes the correct xmlstarlet executable on Debian systems.
   Patch: fix-flymake-xmlstarlet-invocation.diff
   Provided-by: Jussi Judin <jjudin+debian@iki.fi>
   Date: Sat, 20 Oct 2007 14:42:02 UTC
   Added-by: Rob Browning <rlb@defaultvalue.org>
   Status: Debian specific
   Bug: 447378

  Emacs invokes xmlstarlet rather than xml, which is the correct
  executable name on Debian systems.
---
 lisp/progmodes/flymake.el |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lisp/progmodes/flymake.el b/lisp/progmodes/flymake.el
index 2a19821..0fdfe54 100644
--- a/lisp/progmodes/flymake.el
+++ b/lisp/progmodes/flymake.el
@@ -1759,7 +1759,7 @@ Use CREATE-TEMP-F for creating temp copy."
 
 ;;;; xml-specific init-cleanup routines
 (defun flymake-xml-init ()
-  (list "xml" (list "val" (flymake-init-create-temp-buffer-copy 'flymake-create-temp-inplace))))
+  (list "xmlstarlet" (list "val" (flymake-init-create-temp-buffer-copy 'flymake-create-temp-inplace))))
 
 (provide 'flymake)
 
-- 
1.7.4.1

